src/renderers/dom/client/ReactDOMIDOperations.js

```diff
@@ -92,7 +92,6 @@ var ReactDOMIDOperations = {
 };
 
 ReactPerf.measureMethods(ReactDOMIDOperations, 'ReactDOMIDOperations', {
```

Comment:
Oh I guess we were already doing this `measureMethods` call and now we just have more.